src/hw/getOnboardingStatus.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-bitwise */
+import Transport from "@ledgerhq/hw-transport";
+import getVersion from "./getVersion";
+import type { OnboardingInfo, SeedSize } from "../types/onboarding";
+
+// Cf https://ledgerhq.atlassian.net/wiki/spaces/FW/pages/3258089508/Synchronous+onboarding
+const onboardingFlag = 0x04;
+const recoveryModeFlag = 0x01;
+const seedRecoveryFlag = 0x80;
+const seedSizeFlag = 0x60;
+const currentWordFlag = 0x1f;
+const confirmingWordsFlag = 0x01;
```

Comment:
same mask for recovery and confirming? 🤔

Reply:
yes but not on the same Byte

Reply:
just making sure